Documentation/contributor-guide/contributor-membership.md

```diff
@@ -0,0 +1,151 @@
+# Community membership
+
+This doc outlines the various responsibilities of contributor roles in etcd. 
+
+| Role       | Responsibilities                             | Requirements                                                  | Defined by                           |
+|------------|----------------------------------------------|---------------------------------------------------------------|--------------------------------------|
+| Member     | Active contributor in the community          | Sponsored by 2 reviewers and multiple contributions           | etcd GitHub org member               |
+| Reviewer   | Review contributions from other members      | History of review and authorship                              | [MAINTAINERS] file reviewer entry    |
+| Maintainer | Set direction and priorities for the project | Demonstrated responsibility and excellent technical judgement | [MAINTAINERS] file maintainers entry |
+
+## New contributors
+
+New contributors should be welcomed to the community by existing members,
+helped with PR workflow, and directed to relevant documentation and
+communication channels.
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the
+principles in this document, familiarity with project organization, roles,
+policies, procedures, conventions, etc., and technical and/or writing ability.
+Role-specific expectations, responsibilities, and requirements are enumerated
+below.
+
+## Member
+
+Members are continuously active contributors in the community.  They can have
+issues and PRs assigned to them. Members are expected to remain active 
+contributors to the community.
+
+**Defined by:** Member of the etcd GitHub organization
+
+### Requirements
+
+- Have made multiple contributions to the project or community.  Contribution may include, but is not limited to:
+    - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+    - Filing or commenting on issues on GitHub
+    - Contributing to community discussions (e.g. meetings, Slack, email discussion
+      forums, Stack Overflow)
+- Subscribed to [etcd-dev@googlegroups.com]
+- Have read the [contributor guide]
+- Sponsored by one active maintainer.
+    - Sponsors must be from multiple member companies to demonstrate integration across community.
```

Comment:
Not sure if this line make sense if we require **one** active maintainer. 

In the header you had "2 reviewers" and than it made sense.

Reply:
+1

Reply:
Done